lib/netconf/static.go

```diff
@@ -40,6 +43,10 @@ var statics = map[ID]Static{
 	Staging: {
 		OmniExecutionChainID: 16561,
 	},
+	Testnet: {
+		AVSContractAddress:   common.HexToAddress("0xa7b2e7830C51728832D33421670DbBE30299fD92"),
```

Comment:
does the create3-computed address not work here?

Reply:
It didn't work for the Monitor. But in the monitor config we don't go through all of the steps. I merge this as a workaround. This should be revisited/refactored later.